packages/kbn-dev-utils/certs/ca.crt

```diff
@@ -1,20 +1,29 @@
+Bag Attributes
+    friendlyName: elasticsearch
+    localKeyID: 54 69 6D 65 20 31 35 37 34 34 33 38 30 39 34 35 37 35 
+Key Attributes: <No Attributes>
+Bag Attributes
+    friendlyName: ca
+    2.16.840.1.113894.746875.1.1: <Unsupported tag 6>
+subject=/CN=Elastic Certificate Tool Autogenerated CA
+issuer=/CN=Elastic Certificate Tool Autogenerated CA
```

Comment:
These bag attributes are included because the certificate was generated as part of a P12 key store, then exported from that key store using OpenSSL. Any code that reads a PEM-formatted certificate or key will ignore this.